src/sinks/datadog/logs.rs

```diff
@@ -41,7 +41,7 @@ impl SinkConfig for DatadogLogsConfig {
             let tls = if let Some(tls) = &self.tls {
                 Some(TlsSettings::from_options(&Some(tls.clone()))?)
             } else {
-                None
+                Some(TlsSettings::default())
```

Comment:
Unless I'm mistaken, with this change `tls` can never be `None`, so some of this can be streamlined.

BTW This also implies that TLS is always enabled, not that it defaults to on.

Reply:
Yeah, the issue here is if the docs say `tls.enabled = true` is the default, having this as none would be wrong. Right?

Reply:
This much is correct. However, what I am saying is that there is no path in the code where `tls = None`. As such, it's not just the default, it's not possible to disable TLS. You would need to change the `TlsOptions` in the config to `TlsConfig` (which would probably better be named `OptionalTlsConfig` or `TlsConfigWithEnable`).

Reply:
I guess you can still disable tls by passing `enabled = false` right?

Reply:
Try it. I don't think so.